pyproject.toml

```
@@ -116,6 +117,7 @@ ignore = [
   "E221", # Multiple spaces before operator
   "E226", # Missing whitespace around arithmetic operator
   "E241", # Multiple spaces after ','
+  "PYI034", # flake8-pyi: typing.Self added in Python 3.10
```

```suggestion
  "PYI034", # flake8-pyi: typing.Self added in Python 3.11
```
See https://docs.python.org/3/library/typing.html#typing.Self